### PR TITLE
Added option

### PR DESCRIPTION
**Pull request by SMoscarda**
Added option to eat marshmallow after sandwich @casey-collab

### CODE REVIEW

File: swedish/titta-ut/macka/macka.md [hunk 1]
```diff
@@ -1,3 +1,9 @@
 Du går till köket för att göra dig en macka. Väl där hittar du 
 samtliga ingredienser som behövs och äter din nyligen förberedda 
-delikatess i lugn och ro.
\ No newline at end of file
+delikatess i lugn och ro.
+
+När mackan väl landat i magen känner du ett enormt sötsug. På bordet framför dig ligger en marshmallow. Vad gör du?
+
+[Äter en marshmallow.](marshmallowen/marshmallowen.md)
```
Comment (casey-collab) [inline swedish/titta-ut/macka/macka.md]: It looks like this link is broken.  You can view the page on GitHub on your fork and click the link there to check if it is working correctly.

File: swedish/titta-ut/macka/macka.md [hunk 2]
```diff
@@ -1,3 +1,9 @@
...
+
+[Lägger dig för att sova igen.](../start.md)
```
Comment (casey-collab) [inline swedish/titta-ut/macka/macka.md]: This link is also invalid.

**Comment by casey-collab** [comment]
Besides my other comments, this looks good!  Once you respond to those, this should be good to merge.

**Comment by SMoscarda** [comment]
Review this please, @casey-collab

File: swedish/titta-ut/macka/macka.md (hunk 1 above)
Comment (casey-collab) [inline swedish/titta-ut/macka/macka.md]: It looks like this link is broken.  You can view the page on GitHub on your fork and click the link there to check if it is working correctly.

File: swedish/titta-ut/macka/macka.md (hunk 2 above)
Comment (casey-collab) [inline swedish/titta-ut/macka/macka.md]: This link is also invalid.

**Comment by casey-collab** [comment]
Besides my other comments, this looks good!  Once you respond to those, this should be good to merge.

**Comment by SMoscarda** [comment]
Ok, please give it a try @casey-collab.

**Comment by casey-collab** [comment]
Looks great! :checkered_flag:

**Pull request merged**